Browser differences: Note PDF annotations is disabled (#1384)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/concepts/browser-features.md

| Immersive Reader | Off | No | This feature depends on the browser UI for interaction.  This feature is turned off.  |  
| Intrusive Ads | Off | No | This feature is turned off.  |  
| Keyboard Shortcuts | Review Details | Review Details | The keyboard shortcuts that are turned off by default either don't make sense or cause problems in WebView2.  You may not turn on or off these shortcuts.  Instead, you may listen for a key combination using the `AcceleratorKeyPressed` event and create a custom response if needed.  For more information, navigate to [Additional keyboard shortcuts information](#additional-keyboard-shortcuts-information). | 

> | PDF Annotations | Off | No | This feature is turned off. The PDF viewing feature is enabled. But the ability to draw, ink, and highlight in a PDF are disabled. For more information, navigate to [Disabling Feature: PDF Annotations Support](https://github.com/MicrosoftEdge/WebView2Announcements/issues/21). |

Reviewer: ```suggestion
| PDF Annotations | Off | No | This feature is turned off. The PDF viewing feature is enabled, but drawing, inking, and highlighting in a PDF are not enabled. For more information, navigate to [Disabling Feature: PDF Annotations Support](https://github.com/MicrosoftEdge/WebView2Announcements/issues/21). |
```
An Edit pass.

> | PDF Annotations | Off | No | This feature is turned off. The PDF viewing feature generally is on, but specifically the ability to draw, ink, and highlight on a PDF is disabled. For more information, navigate to [Disabling Feature: PDF Annotations Support](https://github.com/MicrosoftEdge/WebView2Announcements/issues/21). |

Reviewer 2: ```suggestion
| PDF Annotations | Off | No | This feature is turned off. The PDF viewing feature is enabled by default. But the ability to draw, ink, and highlight in a PDF are disabled. For more information, navigate to [Disabling Feature: PDF Annotations Support](https://github.com/MicrosoftEdge/WebView2Announcements/issues/21). |
```

Author: I'd like to leave out 'by default' since we don't have a way to disable the PDF viewer feature, but otherwise take the suggestion, thanks!
